CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## CraftDocker 1.1.14 ##
```

Comment:
In that case, it is not... Craft release bugfixes, improvements and new features in their `patch` releases